plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -35,6 +35,11 @@ export const SettingsPaymentsMain = () => {
 	const [ livePaymentsModalVisible, setLivePaymentsModalVisible ] =
 		useState( false );
 
+	const [ storeCountry, setStoreCountry ] = useState< string | null >(
+		window.wcSettings?.admin?.woocommerce_payments_nox_profile?.location ||
+			'US'
```

Comment:
Instead of hardcoding 'US', we need to display the default WooCommerce country.

Reply:
Ah yes, I forgot to change it! On it...

Reply:
Added in 67a8b89f575e507e9e9f47f63d9260f3e1aeb6d7 and updated testing instructions.